server/runtime/src/main/server/server/conf/infinispan-dev-mode.xml

```diff
@@ -0,0 +1,83 @@
+<infinispan
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xsi:schemaLocation="urn:infinispan:config:15.0 https://infinispan.org/schemas/infinispan-config-15.0.xsd
+                            urn:infinispan:server:15.0 https://infinispan.org/schemas/infinispan-server-15.0.xsd
+                            urn:org:jgroups http://www.jgroups.org/schema/jgroups-5.2.xsd"
+      xmlns="urn:infinispan:config:15.0"
+      xmlns:ispn="urn:infinispan:config:15.0"
+      xmlns:server="urn:infinispan:server:15.0">
+
+   <jgroups>
+      <!--
+      This configuration demonstrates cross-site replication on a single host and uses IP multicast for discovery.
+
+      To use this configuration, do the following:
+
+      * Copy the server/ directory to server2/.
+      * Open a terminal and run: ./bin/server.sh -c infinispan-xsite.xml
+      * Open another terminal and run: ./bin/server.sh -c infinispan-xsite.xml -s server2 -Dinfinispan.site.name=site2 -o 100 -Djgroups.mcast_port=46656
+      * Check server logs for the following message: "[org.infinispan.XSITE] ISPN000439: Received new x-site view: [site1, site2]"
+      -->
```

Comment:
Should we automate this for the user so that they can just do something like `./bin/server.sh --xsite-dev`?

Reply:
For now, is mostly for us, so I won't bother much at least for now